addons/survey/report/survey_templates.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="certification_report_view_general">
+            <!-- Style classes to be applyed to '#o_survey_certification': [no class](purple), gold, blue -->
+            <div id="o_survey_certification" t-att-data-oe-model="user_input._name" t-att-data-oe-id="user_input.id" t-att-class="'article certification-wrapper %s %s' % (layout_color,layout_template)">
+                <div class="certification">
+                    <div class="certification-seal" t-if="user_input.scoring_success and layout_template == 'modern'"/>
+                    <div class="certification-top">
+                        <h1><b>Certificate</b>
+                            <br/><span t-if="user_input.scoring_success">of achievement</span>
+                        </h1>
+                    </div>
+
+                    <div class="page certification-content">
+                        <div class="oe_structure"/>
+                        <div t-if="user_input.scoring_success">
+                         <div class="oe_structure"/>
+                            <p> <span>This certificate is presented to</span>
+                                <br/>
+                                <t t-set="certif_style" t-value="''"/>
+                                <t t-set="certified_name" t-value="user_input.partner_id.name or user_input.email"/>
```

Comment:
Maybe you'd like to know and think about the discussion here: https://github.com/odoo/odoo/pull/113459.